src/Concordium/Client/Runner/Helper.hs

```diff
@@ -117,12 +118,17 @@ toGRPCResult' =
                             let hs = map (\(hn, hv) -> (CI.mk hn, hv)) hds
                             in  StatusOk (GRPCResponse hs t)
 
+data Retry a
```

Comment:
Let's put a few words on this and what justifies it's existent.

Reply:
Added